lib/features/composer/presentation/composer_controller.dart

```diff
@@ -166,6 +209,66 @@ class ComposerController extends GetxController {
     return emailQuotedHtml;
   }
 
+  Future<Email> generateEmail() async {
+    final generateEmailId = EmailId(Id(Uuid().v1()));
+    final currentMailboxId = composerArguments.value!.currentMailboxId;
+    final listFromEmailAddress = {
+      EmailAddress(
+        composerArguments.value!.userProfile.getNameDisplay(),
+        composerArguments.value!.userProfile.getEmailAddress())
+    };
+    final generatePartId = PartId(Uuid().v1());
+    final generateBlobId = Id(Uuid().v1());
+
+    final emailBodyText = await htmlEditorController.getText();
+
+    return Email(
+      generateEmailId,
+      mailboxIds: {currentMailboxId: true},
```

Comment:
no, when you compose a new email, it should be in `Outbox`, and then when success, it should be in `Sent`. please read the story: https://github.com/linagora/tmail-flutter/issues/43